lms/templates/courseware/progress.html

```diff
@@ -118,7 +121,7 @@ <h2 class="title">${_("Congratulations, you qualified for a certificate!")}</h2>
                 %elif credit_course_requirements['eligibility_status'] == 'partial_eligible':
                     <span>${student.get_full_name()}, ${_("You have not yet met the requirements for credit.")}</span>
                 %endif
-                <button class="credit-help"><i class="fa fa-question"></i><span class="sr">Help regarding credit requirement</span></button><br>
+                <button class="credit-help" onclick="redirect_to_credit_help()"><i class="fa fa-question"></i><span class="sr">Help regarding course credit requirements</span></button><br>
                 <div class="requirement-container" data-eligible="${credit_course_requirements['eligibility_status']}">
```

Comment:
Could you use <code> <a href="${settings.CREDIT_HELP_LINK_URL}"> </code> instead of ```button```